Doc/library/enum.rst

```diff
@@ -573,6 +588,17 @@ Data Types
       better support the *replacement of existing constants* use-case.
       :meth:`__format__` was already :func:`int.__format__` for that same reason.
 
+.. class:: ReprEnum
+
+   *ReprEum* uses the *repr()* of *Enum*, but the *str()* of the mixed-in data type:
```

Comment:
```suggestion
   :class:`!ReprEum` uses the :meth:`repr() <Enum.__repr__>` of :class:`Enum`,
   but the :class:`str() <str>` of the mixed-in data type:
```

Add standard Sphinx syntax—FYI, `!` keeps the same consistent styling of a class (or other name) but suppresses the (undesired) reference link